src/policy/fees.h

```diff
@@ -179,8 +179,8 @@ static const double MIN_SUCCESS_PCT = .95;
 static const double SUFFICIENT_FEETXS = 1;
 
 // Minimum and Maximum values for tracking feerates
-static constexpr double MIN_FEERATE = 10;
-static const double MAX_FEERATE = 1e7;
+static constexpr double MIN_FEERATE = 1e6;
+static const double MAX_FEERATE = 1e11;
```

Comment:
Could you please express fee parameters relative to `COIN` (or another fee-related parameter)? That was your own request on other policy changes...

Reply:
Done